fern/docs/pages/api-reference-overview.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: API reference
+subtitle: Interactive documentation generated from your API specification
+slug: api-reference
+---
+
+Fern automatically generates interactive API reference documentation from your API specification. This starter template includes a sample Plant Store API to demonstrate the feature.
+
+## Supported specifications
+
+Fern supports multiple API specification formats:
+
+| Format | Description | Documentation |
+|--------|-------------|---------------|
+| **OpenAPI** | The most widely used REST API specification format (versions 3.0 and 3.1) | [OpenAPI docs](https://buildwithfern.com/learn/api-definitions/openapi/overview) |
+| **AsyncAPI** | For event-driven and message-based APIs | [AsyncAPI docs](https://buildwithfern.com/learn/api-definitions/asyncapi) |
+| **gRPC** | Protocol buffer definitions for gRPC services | [gRPC docs](https://buildwithfern.com/learn/api-definitions/protobuf) |
+| **Fern Definition** | Fern's native YAML-based API definition format | [Fern Definition docs](https://buildwithfern.com/learn/api-definitions/fern-definition/overview) |
```

Comment:
Let's remove. Don't want to advertise this as customers don't adopt it highly.

Down to keep gRPC (though less than 5% of users care about it)